salt/client/ssh/__init__.py

```diff
@@ -215,7 +215,7 @@ class SSH(object):
     def __init__(self, opts):
         self.__parsed_rosters = {SSH.ROSTER_UPDATE_FLAG: True}
         pull_sock = os.path.join(opts['sock_dir'], 'master_event_pull.ipc')
-        if os.path.isfile(pull_sock) and HAS_ZMQ:
+        if os.path.exists(pull_sock) and HAS_ZMQ:
```

Comment:
Hmm, that's interesting. Why was this change necessary?

Reply:
So, this never actually worked, os.path.isfile does not return true on sockets, so we were never firing these events on the event bus in the past.

That is why I just threw away the old event that would be thrown and changed it to be the same format as our regular minion events.